substrate/client/cli/src/commands/run_cmd.rs

```diff
@@ -399,6 +407,22 @@ impl CliConfiguration for RunCmd {
 		Ok(self.rpc_max_subscriptions_per_connection)
 	}
 
+	fn rpc_buffer_capacity_per_connection(&self) -> Result<u32> {
```

Comment:
I noticed that this was missing :(

Reply:
Ooh this is the line I missed when I was looking at the PR and only noticed the batch config bits :)